packages/wallet-sdk/src/util/cipher.test.ts

```diff
@@ -56,7 +56,7 @@ describe('Cipher', () => {
 
       // Attempting to decrypt with a different shared secret
       await expect(decrypt(sharedSecretDerivedByPeer, encryptedMessage)).rejects.toThrow(
-        'Unsupported state or unable to authenticate data'
+        'The operation failed for an operation-specific reason'
```

Comment:
What prompted the change in error messaging here? I don't see a related change. Looks like `The operation failed for an operation-specific reason` comes out of the webcrypto api

Reply:
Think this would have been coming from the WebCrypto polyfill, which I removed.

Reply:
That makes sense.